src/phoenix/core/traces.py

```diff
@@ -36,17 +36,17 @@ def __init__(self) -> None:
 
     def get_project(self, project_name: str) -> Optional["Project"]:
         with self._lock:
-            return self._projects.get(project_name)
+            return self._projects.get(project_name, Project())
 
     def get_projects(self) -> Iterator[Tuple[int, str, "Project"]]:
         with self._lock:
             for project_id, (project_name, project) in enumerate(self._projects.items()):
                 if project.is_archived:
                     continue
-                yield project_id, project_name, project
+                yield project_id + 1, project_name, project
```

Comment:
I don't follow the plus 1. shouldn't this be the ID in the db?

Reply:
This is the legacy data structure that is going to be removed, but for the time being we need to make it 1-base index to match the primary keys from database.